ports/raspberrypi/common-hal/microcontroller/Pin.c

```diff
@@ -66,15 +82,30 @@ void reset_pin_number(uint8_t pin_number) {
     hw_set_bits(&padsbank0_hw->io[pin_number], PADS_BANK0_GPIO0_OD_BITS);
 }
 
+void reset_pin_number_cyw(uint8_t pin_no) {
+    cyw_pin_claimed &= ~(1 << pin_no);
+}
+
```

Comment:
Guard with `CIRCUITPY_CYW43` so it's completely not there if compiling for regular Pi Pico?